server/routes/pubDocument.tsx

```diff
@@ -81,12 +85,24 @@ const getEnrichedPubData = async ({
 		getSubmissions: true,
 		getDraft: true,
 		getDiscussions: true,
+		isReview,
 	});
 
 	if (!pubData) {
 		throw new ForbiddenError();
 	}
 
+	if (!pubData.reviewHash) {
```

Comment:
Hmm, I dunno about this. It's more typical to write a migration to backfill these instead. (I appreciate the attention to detail, but if we took this approach everywhere our code would be thick with these ad-hoc backfills by now!)